daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/dpath/NodeInfo.scala

```diff
@@ -298,36 +279,36 @@ object NodeInfo extends Enum {
 
   def fromObject(a: Any) = {
     a match {
-      case x: String => NodeInfo.String
```

Comment:
This could be reverted. 

I was thinking of removing NodeInfo's aliases for all the PrimType objects, but it was too much trouble.